ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -17,6 +17,11 @@ cat << EOF > ~/inventory
 sshd.bastion-telco ansible_ssh_user=tester ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" ansible_password=$SSH_PASS
 EOF
 
+KCLI_PARAM=""
+if [ ! -z $OO_CHANNEL ] ; then
+    KCLI_PARAM="-P openshift_image=registry.svc.ci.openshift.org/ocp/release:$OO_CHANNEL"
```

Comment:
contemplating if it'll be better to include that inside the kcli_parametrs.yml
and use envsubst to replace with real value.
then you can have the default dictated from here.

Reply:
I can put an else here and set the image to some default image if you'd like